auth.go

```diff
@@ -24,6 +24,8 @@ const (
 	sessionTokenLength = 30
 
 	defaultSessionExpirySeconds = 30 * 24 * 3600
+
+	ldapMergeTickerSeconds = 5 * time.Second
```

Comment:
This seems way too frequent. I would imagine every minute is more reasonable. However, for the sake of unit tests, I'm guessing this needs to be configurable. Once again, don't make it a global variable. Make the default a global variable, but make it a configurable member of Central.